src/software/simulation/ssl_simulation_protocol/ssl_simulation_protocol_network_module.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+class SslSimulationProtocolNetworkModule
```

Comment:
Is the file supposed to be a part of this PR? Ditto for the empty files above

Reply:
Oops, removed